src/Orc.Controls/Controls/StepBar/ViewModels/StepBarViewModel.cs

```diff
@@ -0,0 +1,95 @@
+﻿namespace Orc.Controls
+{
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using Catel.MVVM;
+
+    public class StepBarViewModel : ViewModelBase
+    {
+        #region Fields
+        private int _currentIndex = 0;
+        #endregion Fields
+
+        #region Constructors
+        public StepBarViewModel()
+        {
+            QuickNavigateToItem = new TaskCommand<IStepBarItem>(QuickNavigateToItemExecuteAsync, QuickNavigateToItemCanExecute);
```

Comment:
Remove this command. We'll implement a command *per IStepBarItem* so we can implement a command per item if needed (spoiler alert: we'll need that :) )